content/source/docs/enterprise/admin/active-active.html.md

```diff
@@ -0,0 +1,114 @@
+
+Example upload structure
+
+```bash
+support-bundles
+└── 2020-11-10T02:03:05Z
+    ├── 10.0.0.5
+    │   └── replicated-support702524260.tar.gz
+    └── 10.0.0.6
+        └── replicated-support577188727.tar.gz
+```
+
+
+#### node-drain
+
+```bash
+tfe-admin node-drain 
+```
+
+This command will quiesce the current node and remove it from service. It will allow current work to complete and safely stop the node from picking up any new jobs from the Redis queue, allowing the application to be safely stopped. Currently, it only affects `localhost` (it does not support running on one node to drain other nodes). Currently, there is no reverse drain command - a restart is needed to restore the node.
+
+
+####  app-config
+
+```bash
+tfe-admin app-config -k <KEY> -v <VALUE>
+```
+
+This command allows users to make ad hoc/realtime application changes, such as `capacity_concurrency` via the CLI. Both an allowable `<KEY>` (setting name) and `<VALUE>` (new setting value) must be provided. A complete list of the current `app-config` settings can be found by running `replicatedctl app-config export`. You will be warned to restart the TFE application after running this command and must do so with a `replicatedctl app restart` command for the configuration changes to be in effect.
+
+-> **Note:** You should ensure that any ad hoc changes made in this fashion are captured in the standard node build configuration, as the next time you build/rebuild a node only the configuration stored for that purpose will be in effect and ad hoc changes could be lost.
+
+-> **Hint:** Adding a function to your Linux start-up like an alias can give you a short cut to the admin `app-config` command only requiring a single command and parameters, such as:
+
+```bash
+# shortcut: tfe-app-config <KEY> <VALUE>
+tfe-app-config ()
+{
+        tfe-admin app-config -k "$1" -v "$2"
+}
+```
+
+## Other Supporting Commands
+
+There are additional commands available for checking status and troubleshooting directly on nodes. You can use them to confirm successful installation or to check on the status of a running node as part of troubleshooting activities.  Also, there are additional command aliases available that allow you to run more abbreviated versions of commands like just `support-bundle`. Run an `alias` command with no parameters to see the list of available command aliases.
+
+### Commands
+
+#### health-check
+
+```bash
+ptfe-admin health-check
+(alias health-check)
+
+```
+
+This command tests and reports on the status of the major TFE services. Each will be listed as PASS or FAIL. If any are marked as FAIL, your TFE implementation is NOT healthy and additional action must be taken.
+
+
+#### replicated status
+
+```bash
+replicatedctl system status
```

Comment:
This, like all replicatedctl commands, is per machine. There's some work that exists to bring this into tfe-admin to aggregate these responses.

Reply:
clarified